x-pack/plugins/security_solution/public/resolver/lib/date.ts

```diff
@@ -77,6 +77,6 @@ export const getFriendlyElapsedTime = (
       break;
   }
 
-  const durationType = duration > 1 ? pluralType : singularType;
-  return { duration: Math.floor(duration), durationType };
+  const durationType = duration === 1 ? singularType : pluralType;
```

Comment:
:grey_question: Do we want something like "<1 ms" if it's less than 1? So it doesn't show "0ms"? Or maybe leave it blank if it's actually zero or something...

Reply:
@lindseypoli ?

Reply:
I like Brent's idea of using "<1 millisecond".

"0 milliseconds" is a little confusing and leaving it blank could look like an error/bug.

Reply:
Sounds good. Fixed :) 

![image](https://user-images.githubusercontent.com/17211684/88309172-f7789380-ccdb-11ea-87aa-5519de52cb95.png)